lib/ecco/row_converter.rb

```diff
@@ -0,0 +1,61 @@
+module Ecco
+  module RubyConverter
```

Comment:
I just noticed now that the file is `row_converter` but the module is called `RubyConverter`. I should probably fix this so that it is cohesive somehow.

Reply:
Yeah, `RowsConverter` would make sense to me at least. (so file would be `rows_converter.rb`)

Reply:
Fixed here: 5f2d1e5735964f985c46fd85b71527fda9a5cca8